codes/classical/q-ary_digits/hermitian_codes.yml

```diff
@@ -0,0 +1,62 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: hermitian_code
+physical: q-ary
+logical: q-ary
+name: 'Hermitian Codes'
+introduced: '\cite{manual: {Goppa, Valerii Denisovich. "Codes on algebraic curves." Soviet Math. Dokl.. Vol. 24. 1981.}}'
+
+description: | 
+  A Hermitian code is a type of algebraic geometry code where the codewords are evaluations of two-variable polynomials of total degree at most \(D\) on 
+  the rational points of a Hermitian curve \(H(X,Y) = X^{q+1}  + Y^{q+1} - 1\). These evaluations may be made in affine coordinates \( (X,Y) \) or 
+  over projective coordinates \( [X:Y:Z] \) which contain as a subset the affine coordinates (this is the case when \(Z = 1 \)). In this entry we focus on 
+  evaluations over projective coordinates, since doing so increases the length of the code. More precisely, fix \(r, D\) and let 
```

Comment:
If evaluating on the projective points, we also need homogeneous (trivariate) message polynomials, or else we have no way to substitute points [X:Y:Z] into them.